### PR TITLE
HTML-CSS Popup

### PR DESCRIPTION
**Pull request by v4voloshyn**
### HTML-CSS Popup

[Demo](https://v4voloshyn.github.io/popup-kottans/) |
[Code base](https://github.com/v4voloshyn/popup-kottans)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by v4voloshyn** [comment]
I'm pretty sure that it's everything alright with my code. But nobody is perfect :)

P.S. I found some bug in my code, but can do you? 😁

File: submissions/v4voloshyn/html-css-popup/styles.css [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+	height: 100%;
+	display: flex;
+	align-items: center;
+	border-radius: 5px;
+	transition: all ease-in-out .3s;
+}
+
+.nav-link {
+	font-size: 18px;
+	text-transform: capitalize;
```
Comment (kasionio) [inline submissions/v4voloshyn/html-css-popup/styles.css:78]: seems you did it manually :)

File: submissions/v4voloshyn/html-css-popup/styles.css [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+	grid-auto-flow: row;
+	grid-template-columns: repeat(3, 95px);
+	grid-auto-rows: 1fr;
+	justify-content: center;
+	column-gap: 5px;
+	row-gap: 5px;
+	box-shadow: rgba(55, 52, 52, 0.5) 0 0 20px;
+	transition: all ease-in .3s;
+	padding-right: 0;
+	/* experemental feature >| */
```
Comment (kasionio) [inline submissions/v4voloshyn/html-css-popup/styles.css:127]: this is a good comment 👍

File: submissions/v4voloshyn/html-css-popup/styles.css [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+.more+.more-label:hover::before,
+.more:focus+.more-label:before {
+	opacity: 1;
+}
+
+.more:focus+.more-label,
+.more:hover+.more-label {
+	border: 2px solid black;
+	box-shadow: rgb(173, 20, 110) 0px 5px 10px;
```
Comment (kasionio) [inline submissions/v4voloshyn/html-css-popup/styles.css:323]: first dimensions, them color

File: submissions/v4voloshyn/html-css-popup/styles.css [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+	cursor: pointer;
+	display: flex;
+	border-radius: 15px;
+	width: 40px;
+	height: 40px;
+	align-items: center;
+	transition: border ease-in-out .3s;
+}
+
+.popup-list {
```
Comment (kasionio) [inline submissions/v4voloshyn/html-css-popup/styles.css:107]: one class and so many properties 😞  Please try to optimize your code, it is not easy to review. You can refactor and simplify it, some of the properties are redundant

**Review by kasionio — CHANGES_REQUESTED** [review]
Nice job, good design! Try to make a self-review and optimize the code ;) And take a look at the comments below

**Pull request closed without merging**